stories/examples/Title.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { ThemeProvider, compose } from 'react-themed'
+import { Title } from 'react-ui-core/src/Title'
+import { TitleTheme } from '../theme'
+
+const theme = compose(
+  {},
+  TitleTheme,
+)
+
+export const DefaultTitle = (
+  <ThemeProvider theme={theme}>
+    <Title
+      nodeType="h2"
```

Comment:
If this is `DefaultTitle` you probably shouldn't pass `nodeType` and leave that for the other story.

Reply:
Good point - I made the changes and updated the children propType to accommodate it.

Reply:
You should still not pass the `nodeType` in this example as it looks like you are not using the default behavior in the default example.